src/gatsby-ssr.js

```diff
@@ -83,7 +83,7 @@ exports.onRenderBody = (
         key: 'gatsby-remark-klipse-codemirror',
         rel: 'stylesheet',
         src:
-          'https://storage.googleapis.com/app.klipse.tech/css/codemirror.css',
+          'https://unpkg.com/klipse@7.5.0/dist/codemirror.css',
```

Comment:
Maybe we can add a `version` option & we fallback to this if `version` is not passed, this way we don't need to update it every time when klipse updates to a new version.